proto/asn1-pdu/mutated_x509_chain_to_der.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#ifndef PROTO_ASN1_PDU_MUTATE_X509_CHAIN_H_
+#define PROTO_ASN1_PDU_MUTATE_X509_CHAIN_H_
+
+#include <stdint.h>
+
+#include <vector>
+
+#include "mutated_x509_chain.pb.h"
+
+namespace mutated_x509_chain {
```

Comment:
Mentioned on Friday, but don't put this in a separate namespace :) Recall we talked about just making this part of the x509_certificate namespace, because logically, this is part of mutating chains.